commands/delete.go

```diff
@@ -34,18 +36,22 @@ hub-init(1), hub(1)
 `,
 }
 
-var ()
+var (
+	flagDeleteAssumeYes bool
+)
 
 func init() {
+	cmdDelete.Flag.BoolVarP(&flagDeleteAssumeYes, "--yes", "y", false, "YES")
+
 	CmdRunner.Use(cmdDelete)
 }
 
 func delete(command *Command, args *Args) {
 	var repoName string
 	if args.IsParamsEmpty() {
-		dirName, err := git.WorkdirName()
-		utils.Check(err)
-		repoName = github.SanitizeProjectName(dirName)
+		ui.Errorln("Expecting name of reposiroty to delete")
```

Comment:
1. Typo in `repository`
2. It would be great if the params empty case simply printed out the command's synopsis on stderr and exited with a non-zero status. https://github.com/github/hub/blob/da7348a83b41bca0b27b58ac356b34c6fc536c83/commands/commands.go#L95-L107

Reply:
ok. done